src/components/Message/__tests__/message.js

```diff
@@ -1,6 +1,6 @@
-import ErrorPage from '../ErrorPage';
+import Message from '../Message';
 
 it('renders without crashing', () => {
```

Comment:
Please provide a test suite by doing the following
```js
describe('message test suite', () => {
  it('should render without crashing', async () => {
    ...
  })
})
```

Also please read [Guide](https://devhints.io/jest)